crates/swift-bridge-ir/src/codegen/generate_swift/generate_function_swift_calls_rust.rs

```diff
@@ -194,33 +194,89 @@ pub(super) fn gen_func_swift_calls_rust(
         let func_ret_ty = function.return_ty_built_in(types).unwrap();
         let rust_fn_ret_ty =
             func_ret_ty.to_swift_type(TypePosition::FnReturn(HostLang::Rust), types);
-
         let (maybe_on_complete_sig_ret_val, on_complete_ret_val) = if func_ret_ty.is_null() {
             ("".to_string(), "()".to_string())
         } else {
-            (
-                format!(
```

Comment:
We can undo this change and go back to the original code.

The `if` and `else` blocks that were added are identical.

Reply:
Done.